Bottom: 4c1e9af7174f58ced0b5b72be7c3c8278afe6684
Top:    524bd26fe227be4eab4b8032597f4ebbf31beade
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-13 01:19:27 +0100

hid/common: Remove unused function hid_register_gui()

GUIs are registered with the same API as exporter HIDs.


---

diff --git a/src/hid/common/hidinit.c b/src/hid/common/hidinit.c
index 554e592..4233b4c 100644
--- a/src/hid/common/hidinit.c
+++ b/src/hid/common/hidinit.c
@@ -176,19 +176,6 @@ hid_register_hid (HID * hid)
   hid_list[hid_num_hids] = 0;
 }
 
-static void (*gui_start) (int *, char ***) = 0;
-static HID *default_gui = 0;
-
-void
-hid_register_gui (HID * Pgui, void (*func) (int *, char ***))
-{
-  if (gui_start)
-    return;
-
-  default_gui = Pgui;
-  gui_start = func;
-}
-
 
 HID *
 hid_find_gui ()
